drivers/sensor/ccs811/ccs811.h

```diff
@@ -85,6 +78,15 @@ struct ccs811_data {
 
 struct ccs811_config {
 	struct i2c_dt_spec i2c;
+#if DT_INST_NODE_HAS_PROP(0, irq_gpios)
+	struct gpio_dt_spec irq_gpio;
+#endif
+#if DT_INST_NODE_HAS_PROP(0, reset_gpios)
+	struct gpio_dt_spec reset_gpio;
+#endif
+#if DT_INST_NODE_HAS_PROP(0, wake_gpios)
+	struct gpio_dt_spec wake_gpio;
+#endif
```

Comment:
this is not multi-instance capable, what if instance 1 has them and not 0? however, this is how current code works so I guess it can be fixed later.

Reply:
That's kind of what I was asking for in the first question here:
https://github.com/zephyrproject-rtos/zephyr/pull/46669#discussion_r901786838

I ended up compromising for clarity and left the multi-instance question for a later pull-request when we update all sensor drivers to be multi-instance capable.

Reply:
Agreed it's better to fix this later